LICENSE.txt

```diff
@@ -1,5 +1,6 @@
 BSD 3-clause license
-Copyright (c) 2015-2020, conda-forge contributors
+Copyright (c) 2015-2022, conda-forge contributors
```

Comment:
Lets not update the conda-forge copyright. It should remain as:
`Copyright (c) 2015-2020, conda-forge contributors`